_posts/1-2-9-DropDownMenu.md

```diff
@@ -53,7 +53,6 @@ DropDownMenu is a full-screen contextual menu for displaying lists of items.
 
 ## Examples
 
-#### JSX declaration
```

Comment:
@tenodi   i would bring back this line, we have this concept everywhere (first `## examples` and below `#### jsx declaration`)

what do you think?